index.html

```diff
@@ -2,9 +2,134 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <title>My Journey with ACA</title>
+    <link type="text/css" rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
   </head>
-  <body>
-    <h1>Hello world!</h1>
-  </body>
+<body>
+  <header> <br><br>
+      <h1>Heather<strong>Barr</strong></h1>
+      <h3>ACA: 101 Intro to Web</h3>
+  </header>
+  <main>
+    <!-- these are my blocks with brief summary on project and a link included -->
+    <section class="week">
+        <h2>Week 01</h2>
+        <h4>Resume</h4>
+        <p>This was my first homeowrk assignment with ACA! During week one, we went over html and were to create a resume using html. I love CSS, so I had to use some.<br></p>
+        <p><em>What I used;</em></p>
+            <li>Basic HTML</li>
+            <li>Basic CSS</li>
+            <li>Column Use</li><br>
+        <button type="button"><a href="https://rawgit.com/hnbarr/web-workbook/resume/01week/resume/index.html">Check it Out!</a></button>
+    </section><br>
+
+    <section class="week">
+          <h2>Week 02</h2>
+          <h4>Bio</h4>
+          <p>After learning more about CSS, we were to create a bio page! I chose to do something with a simple feel but still creative and fun.<br></p>
+          <p><em>What I used;</em></p>
+            <li>Navigation</li>
+            <li>Footer with working links to contact me</li>
+            <li>Hover Action</li><br>
+          <button type="button"><a href="https://rawgit.com/hnbarr/web-workbook/bio/02week/bio/index.html">Take a Look!</a></button>
+          <h4>Landing-Page</h4>
+          <p>This is when CSS started getting a little more complicated! We learned about different types of selectors and were to create a page to house our other projects!<br></p>
+          <p><em>What I Used;</em></p> 
+            <li>Selectors on Selectors on Selectors</li>    
+            <li>Buttons with links to past projects</li><br>
+        <button type="button"><a href="https://rawgit.com/hnbarr/web-workbook/landing-page/index.html">Useless Button!</a></button>
```

Comment:
same comment as above

Reply:
Yeah, I tried opening my folder on my desktop and they're all empty. So I wasn't sure how to do it. In my web-workbook, all the weeks don't have any content.

Reply:
wait. I can try! Hang on!